addons/udes_stock/tests/test_stock_picking_type.py

```diff
@@ -0,0 +1,14 @@
+from . import common
+
+
+class TestStockPickingCommon(common.BaseUDES):
+    def test_writing_to_picking_type(self):
```

Comment:
Can we make this clearer what it is testing please. 
Test writing on picing type does not reset the picking type prefix?